src/test/java/core/basesyntax/service/TransactionGenerateReportServiceImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class TransactionGenerateReportServiceImplTest {
+    private static final String FIRST_LINE_REPORT = "fruit,quantity";
+    private static final String APPLE = "apple";
+    private static final String BANANA = "banana";
+    private static final String SIGN_SEPARATOR = ",";
+    private static final int APPLE_COUNT = 5;
+    private static final int BANANA_COUNT = 7;
+
+    private static final TransactionGenerateReportService generateReportService
+            = new TransactionGenerateReportServiceImpl();
```

Comment:
initialize this service in a method annotated with Before or BeforeClass (depends on the version of junit)